telegram/ext/_contexttypes.py

```diff
@@ -53,6 +53,19 @@ class ContextTypes(Generic[CCT, UD, CD, BD]):
 
     """
 
+    DEFAULT_TYPE = CallbackContext["ExtBot", Dict, Dict, Dict]  # type: ignore[misc]
+    """Shortcut for the type annotation for the `context` argument that's correct for the
+    default settings, i.e. if :class:`telegram.ext.ContextTypes` is not used.
+
```

Comment:
should we maybe still add a comment that this may not work with all type checkers? 🤔

Reply:
Don't think its needed as long we're supporting the popular type checkers